tests/unit/utils/test_archive.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+from typing import Optional
+from zipfile import ZipFile
+
+import pytest
+
+from ggshield.utils.archive import (
+    UnsafeArchive,
+    _zip_get_symlink_target,
+    check_archive_content,
+    safe_unpack,
+)
+from tests.unit.conftest import DATA_PATH
+
+
+ARCHIVES_PATH = DATA_PATH / "archives"
+BAD_ZIP_PATH = ARCHIVES_PATH / "bad.zip"
+BAD_TAR_PATH = ARCHIVES_PATH / "bad.tar"
+GOOD_ZIP_PATH = ARCHIVES_PATH / "good.zip"
+GOOD_WHL_PATH = ARCHIVES_PATH / "good.whl"
+GOOD_TAR_PATH = ARCHIVES_PATH / "good.tar"
```

Comment:
Our documentation says `Supported archive formats are zip, tar, tar.gz,  tar.bz2 and tar.xz.` 
Could it be worth explicitly adding .whl to the doc ? and should we also test good and bad for ` tar.gz,  tar.bz2 and tar.xz` ?